taichi/inc/rhi_constants.inc.h

```diff
@@ -7,6 +7,7 @@
 // should not.
 #ifdef PER_DEVICE_CAPABILITY
 // SPIR-V Caps
+PER_DEVICE_CAPABILITY(reserved)
```

Comment:
Breaking change because the 0 code is better reserved to helps us prefix capability from different backends.